src/pal/src/exception/seh-unwind.cpp

```diff
@@ -141,7 +141,7 @@ static void UnwindContextToWinContext(unw_cursor_t *cursor, CONTEXT *winContext)
 #elif defined(_ARM_)
     unw_get_reg(cursor, UNW_REG_SP, (unw_word_t *) &winContext->Sp);
     unw_get_reg(cursor, UNW_REG_IP, (unw_word_t *) &winContext->Pc);
-    winContext->Pc &= ~0x1;
+    winContext->Pc |= 0x1;
```

Comment:
A few months ago, when Linux/ARM CoreCLR was extremely unstable, ensuring the last bit unset for libunwind-arm helped stabilizing exception handling and GC. However, with much more robust Linux/ARM CoreCLR support, such workaround might be unnecessary. I'll test with some cases.